keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -794,6 +799,11 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 
 - <test>: <link to test coverage>
 
+No tests exist yet, but a combination of e2e and e2e_node tests will be added for beta:
```

Comment:
I don't think this is true. @saschagrunert added a presubmit to test this feature in alpha.

Reply:
https://testgrid.k8s.io/sig-node-cri-o#pr-crio-cgrpv2-imagevolume-e2e

Reply:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/image_volume.go

Reply:
I would also assume that the existing tests are sufficient for now. If we decide to expand the feature, then an e2e test update would be expected as well.

Reply:
e2e are mandatory for betas IIRC